core-request.html

```diff
@@ -64,7 +71,16 @@
        */
       completes: {
         type: Promise,
-        readOnly: true
+        readOnly: true,
+        notify: true,
+        value: function() {
+          var completes = new Promise(function (resolve, reject) {
```

Comment:
No, it wasn't considered. No, it's not absolutely required. It's too bad, given the contemporary prevalence of Promises and the minimal size of standards-compatible Polyfill implementations :disappointed:

Will fix.